### ISSUE TITLE
Combine the cluster build script and the integration tests

### ISSUE THREAD

**Issue by digitalronin** [ministryofjustice/cloud-platform#1050]
Depends on #1048
Related to #1090 

It should be possible to run the build script and, with no further manual intervention, see a report on how well the new cluster satisfies the criteria embodied in the smoke tests.

Stretch goal:

With a single command;

* create a new cluster
* run the smoke tests and send the report somewhere
* destroy the new cluster (but only if the smoke tests came back green)
